.circleci/config.yml

```diff
@@ -12,26 +12,46 @@ var_4: &save_cache
   save_cache:
     key: *cache_key
     paths:
-    - "node_modules/"
+    - "node_modules"
```

Comment:
I personally prefer the trailing slash as it makes it clear that this is a directory and _not_ a file. Not a big deal though 😄

Reply:
😀 I actually made that change to align with what's in `angular/components` 😉

Reply:
Haha, you got me there 😄 No for real, next time I'll change things there I'll make the change there!

Reply:
Sounds good! 😆